File: app/src/main/java/omniex/nl/omniex/ui/app/profile/address/edit/EditAddressView.java
package omniex.nl.omniex.ui.app.profile.address.edit;

import java.util.ArrayList;

import nl.omniex.omniexshopping.data.model.address.Country;
import nl.omniex.omniexshopping.data.model.address.Province;
import nl.omniex.omniexshopping.ui.base.BaseView;

public interface EditAddressView extends BaseView {

    void onCountriesFetched(ArrayList<Country> countries);

    void onZonesFetched(ArrayList<Province> zones);

    void onAddressRemoved();

    void onAddressSaved();
}

File: app/src/main/java/omniex/nl/omniex/ui/app/profile/address/edit/EditAddressView.kt
package omniex.nl.omniex.ui.app.profile.address.edit

import omniex.nl.omniex.data.model.address.Country
import omniex.nl.omniex.data.model.address.Province
import omniex.nl.omniex.ui.base.BaseView
import java.util.ArrayList

interface EditAddressView : BaseView {

    fun onCountriesFetched(countries: ArrayList<Country>)

    fun onZonesFetched(zones: ArrayList<Province>)

    fun onAddressRemoved()

    fun onAddressSaved()
}
